addons/mail/models/mail_thread.py

```diff
@@ -244,15 +247,8 @@ def create(self, values):
 
         # track values
         if not self._context.get('mail_notrack'):
-            if 'lang' not in self._context:
-                track_thread = thread.with_context(lang=self.env.user.lang)
-            else:
-                track_thread = thread
-            tracked_fields = track_thread._get_tracked_fields(list(values))
-            if tracked_fields:
-                initial_values = {thread.id: dict.fromkeys(tracked_fields, False)}
-                track_thread.message_track(tracked_fields, initial_values)
-
+            initial_values = {thread.id: {}}
+            self.with_context(is_new=1).message_track([], initial_values)
```

Comment:
`tracked_fields` has been replaced by `[]`.
So the tracking functionality is effectively gone here...

Reply:
The info about fields modified is in the message now. (we don't need the mail.tracking.value table anymore). It would be better to remove the feature frmo the JS side too (instead of passing []) but this can be done later.

Reply:
The only problem is helpdesk that has to be rewritten (in Enterprise) as it uses mail.tracking.value to compute a field.